core/docs/spark-profiling-tool.md

```diff
@@ -580,6 +499,90 @@ The _Auto-Tuner_ output has 2 main sections:
   - 'spark.sql.adaptive.enabled' should be enabled for better performance.
   ```
 
+### Generating Visualizations
+
+- Print SQL Plans (--print-plans option):
+Prints the SQL plan as a text string to a file named `planDescriptions.log`.
+For example if your application id is app-20210507103057-0000, then the
+filename will be `planDescriptions.log`
+
+- Generate DOT graph for each SQL (--generate-dot option):
+
+```
+Generated DOT graphs for app app-20210507103057-0000 to /path/. in 17 second(s)
+```
```

Comment:
We might want to note that this generates multiple dot files.  I guess it kind of says that.  When I ran it on a power-run, I got one for each query: `query-0.dot`, etc...

Reply:
OK, i'll add a note.